packages/core/src/renderTexture/RenderTextureSystem.ts

```diff
@@ -73,80 +76,82 @@ export class RenderTextureSystem extends System
 
     /**
      * Bind the current render texture
+     *
      * @param {PIXI.RenderTexture} [renderTexture] - RenderTexture to bind, by default its `null`, the screen
      * @param {PIXI.Rectangle} [sourceFrame] - part of screen that is mapped to the renderTexture
      * @param {PIXI.Rectangle} [destinationFrame] - part of renderTexture, by default it has the same size as sourceFrame
      */
     bind(renderTexture: RenderTexture = null, sourceFrame?: Rectangle, destinationFrame?: Rectangle): void
     {
-        this.current = renderTexture;
-
         const renderer = this.renderer;
 
+        this.current = renderTexture;
```

Comment:
When we use `current` we usualy compare if its the same as before. in RenderTextureSystem we have to bind it anyway, update uniforms and do other stuff. Add a comment about it here please :)

Reply:
You can add a your commit here, no problem 😃.

Reply:
OK, I'm sure you'll add it before the merge ;)

If someone later adds "optimization" here `if (current === renderTexture) return;` that'll trigger several disasters.